test/src/post-make.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+function main() {
+	for use_dir in "${@}"; do
+		# we only want to build libinconsistent.dylib as a fat mach-o
+		if [[ "${use_dir}" == fat* ]]; then
```

Comment:
Best practice is to favor `=` over `==` when doing string comparison in shell code. Another thing you might want to consider that avoids too much nesting is the shell equivalent of `next if` in Ruby, e.g.:

```shell
[[ "${use_dir}" = fat* ]] || continue
inconsistent_for "${use_dir}"
```